test/utils.js

```diff
@@ -1,12 +1,11 @@
 module.exports = {
-  expectThrow: async (promise) => {
-    const errMsg = 'Expected throw not received';
-    try {
-      await promise;
-    } catch (err) {
-      assert(err.toString().includes('invalid opcode'), errMsg);
-      return;
-    }
-    assert.fail(errMsg);
-  },
+  assertRevert: async promise => {
```

Comment:
This is similar to Simon's 721 WIP, so merging that should be a bit less painful.